### PR TITLE
Add new navigation and sub-navigation, updates to footer and brands

### PR DESCRIPTION
**Pull request by craigcook**
This adds new files for redesigned global navigation and sub-navigation. sub-navigation.ftl will be a global Fluent file in bedrock, even though only some pages have subnav; it's more practical to keep the strings consolidated since many get re-used on different pages (also some of those pages aren't converted to Fluent yet).

This also makes a bunch of changes to footer.ftl, but I wonder if it's better to create a footer-v2.ftl? A lot of existing strings are remaining, but there are several new ones and a lot becoming obsolete. Would a new file be cleaner?

Also adding a few new brand names.

Nav is being added to bedrock in https://github.com/mozilla/bedrock/pull/9650. I've use fallback strings from the old nav where possible, and in other cases I'm using `ftl_has_messages` to hide strings until they're translated.

Currently viewable on demo at https://www-demo1.allizom.org

### CODE REVIEW

File: en/navigation_v2.ftl [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+navigation-v2-download-firefox = Download { -brand-name-firefox }
+navigation-v2-get-a-firefox-account = Get a { -brand-name-firefox-account }
+navigation-v2-menu = Menu
+
+## Firefox menu
+
+navigation-v2-firefox-browsers = { -brand-name-firefox } Browsers
+navigation-v2-close-firefox-browsers-menu = Close { -brand-name-firefox } Browsers menu
+navigation-v2-firefox-for-desktop = { -brand-name-firefox } for Desktop
+navigation-v2-get-the-non-profit-backed = Get the non-profit-backed browser on { -brand-name-windows }, { -brand-name-mac-short } or { -brand-name-linux }.
```
Comment (peiying2) [inline en/navigation_v2.ftl:16]: Should we change all `non-profit` to `not-for-profit`? I believe this is the current expression we use.

File: en/navigation_v2.ftl [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+navigation-v2-firefox-for-android = { -brand-name-firefox } for { -brand-name-android }
+navigation-v2-get-the-customizable-mobile = Get the customizable mobile browser for { -brand-name-android } smartphones.
+navigation-v2-firefox-for-ios = { -brand-name-firefox } for { -brand-name-ios }
+navigation-v2-get-the-mobile-browser = Get the mobile browser for your { -brand-name-iphone } or { -brand-name-ipad }.
+navigation-v2-privacy-promise = Privacy Promise
+navigation-v2-learn-how-firefox-treats = Learn how { -brand-name-firefox } treats your data with respect.
+navigation-v2-firefox-blog = { -brand-name-firefox } Blog
+navigation-v2-read-about-new-firefox-features = Read about new Firefox features and ways to stay safe online.
```
Comment (peiying2) [inline en/navigation_v2.ftl:24]: Missed `Firefox`.

File: en/navigation_v2.ftl [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+navigation-v2-release-notes = Release Notes
+navigation-v2-get-the-details-on-the = Get the details on the latest { -brand-name-firefox } updates.
+navigation-v2-view-all-firefox-browsers = View all { -brand-name-firefox } Browsers
```
Comment (peiying2) [inline en/navigation_v2.ftl:27]: do we use `Firefox Browsers` as in the brand file?

File: en/navigation_v2.ftl [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+
+navigation-v2-mozilla = { -brand-name-mozilla }
+navigation-v2-download-firefox = Download { -brand-name-firefox }
+navigation-v2-get-a-firefox-account = Get a { -brand-name-firefox-account }
+navigation-v2-menu = Menu
+
+## Firefox menu
+
+navigation-v2-firefox-browsers = { -brand-name-firefox } Browsers
```
Comment (peiying2) [inline en/navigation_v2.ftl:13]: `Firefox Browsers` as a brand

File: en/navigation_v2.ftl [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+navigation-v2-close-firefox-browsers-menu = Close { -brand-name-firefox } Browsers menu
```
Comment (peiying2) [inline en/navigation_v2.ftl:14]: `Firefox Browsers` as a brand

File: en/navigation_v2.ftl [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+## Products menu
+
+navigation-v2-products = Products
+navigation-v2-close-products-menu = Close Products menu
+navigation-v2-firefox-monitor = { -brand-name-firefox-monitor }
+navigation-v2-see-if-your-email-has = See if your email has appeared in a company’s data breach.
+navigation-v2-pocket = { -brand-name-pocket }
+navigation-v2-save-and-discover-the-best = Save and discover the best stories from across the web.
+navigation-v2-facebook-container = { -brand-name-facebook-container }
+navigation-v2-help-prevent-facebook-from = Help prevent Facebook from collecting your data outside their site.
```
Comment (peiying2) [inline en/navigation_v2.ftl:38]: `Facebook`

File: en/navigation_v2.ftl [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+navigation-v2-view-all-products = View all Products
+
+## Who We Are menu
+
+navigation-v2-who-we-are = Who We Are
+navigation-v2-close-who-we-are-menu = Close Who We Are menu
+navigation-v2-mozilla-manifesto = { -brand-name-mozilla } Manifesto
+navigation-v2-learn-about-the-values = Learn about the values and principles that guide our mission.
+navigation-v2-mozilla-foundation = { -brand-name-mozilla-foundation }
+navigation-v2-meet-the-non-profit-behind = Meet the non-profit behind { -brand-name-firefox } that stands for a better web.
```
Comment (peiying2) [inline en/navigation_v2.ftl:56]: `non-profit`

File: en/navigation_v2.ftl [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+navigation-v2-firefox-monitor = { -brand-name-firefox-monitor }
+navigation-v2-see-if-your-email-has = See if your email has appeared in a company’s data breach.
+navigation-v2-pocket = { -brand-name-pocket }
+navigation-v2-save-and-discover-the-best = Save and discover the best stories from across the web.
+navigation-v2-facebook-container = { -brand-name-facebook-container }
+navigation-v2-help-prevent-facebook-from = Help prevent Facebook from collecting your data outside their site.
+navigation-v2-mozilla-vpn = { -brand-name-mozilla-vpn }
+navigation-v2-get-protection-beyond-your-browser = Get protection beyond your browser, on all your devices.
+navigation-v2-product-promise = Product Promise
+navigation-v2-learn-how-each-firefox-product = Learn how each Firefox product protects and respects your data.
```
Comment (peiying2) [inline en/navigation_v2.ftl:42]: `Firefox`

**Review by peiying2 — CHANGES_REQUESTED** [review]
Let's keep the Footer the way your prepared. Do you know when we want to switch over? Can we remove the outdated strings now or does it look weird (mixed languages). They are not like new strings replacing the old, but straight out old strings that need to be removed. 

A few comments on the other files. It's branding intensive so a few missed ones. Also the `non-profit` expression - it keeps coming up. Let's make sure the writer uses the new expression: `not-for-profit`.

File: en/navigation_v2.ftl (hunk 3 above)
Comment (craigcook) [reply en/navigation_v2.ftl:27]: It's only singular `Firefox Browser` in brands.ftl, unless we want to add a plural form. But the way it's used here I think is correct. This "Browsers" isn't part of the brand name, it's a general term for all our browsers.

Comment (craigcook) [reply en/navigation_v2.ftl:27]: Oh nevermind, we do have the plural. I'll update.

**Review by peiying2 — APPROVED** [review]
Thanks for the quick fixes. I will send out the comms to the communities.

**Comment by craigcook** [comment]
I think most of the outdated strings from the footer can be removed, but not until the new one gets merged to bedrock (that PR still needs review, so hopefully next week). Let's plan to clean up the footer strings later.

**Pull request merged**